highfive/newpr.py

```diff
@@ -269,12 +284,16 @@ def choose_reviewer(repo, owner, diff, exclude, config):
     if exclude in reviewers:
         reviewers.remove(exclude)
 
+    mention_list = None
     if reviewers:
         random.seed()
-        return random.choice(reviewers)
-    else:
-        # no eligible reviewer found
-        return None
+        if len(to_mention) > 0:
```

Comment:
Well, it allows to keep `mention_list = None`. Not absolutely necessary but I like keeping things this way.